controllers/cluster-registration/cluster_registration_controller.go

```diff
@@ -213,7 +224,7 @@ func (r *RegisteredClusterReconciler) updateImportCommand(regCluster *singaporev
 		ImportCommand: importCommand,
 	}
 
-	_, err = r.HubApplier.ApplyDirectly(readerDeploy, values, false, "", files...)
+	_, err = applier.ApplyDirectly(readerDeploy, values, false, "", files...)
```

Comment:
It looks like with this change we're not using RegisteredClusterReconciler.HubApplier at all. Should we just remove this field? Are we missing out on some caching? Is there a way to take advantage of that caching while still using WithOwner? @itdove

Reply:
@robinbobbitt I don't think so here cache is of any use. We are not using Get or List or any other operation on configmap. So no need to cache it.